icepyx/tests/test_variables.py

```diff
@@ -22,3 +43,60 @@ def test_list_of_dict_vals():
     ]
 
     assert obs == exp
+
+def test_avail_from_file(variables_obj_atl06, avail_vars_atl06):
+    assert variables_obj_atl06.avail() == avail_vars_atl06
+
+
+def test_avail_from_order(avail_vars_atl06):
+    region = Query('ATL06',[-45, 74, -44,75],['2019-11-30','2019-11-30'], \
+                       start_time='00:00:00', end_time='23:59:59')
```

Comment:
Recommend to set the `version` here, because the returned schema can change from version to version.
```suggestion
    region = Query(
        product="ATL06",
        spatial_extent=[-45, 74, -44, 75],
        date_range=["2019-11-30", "2019-11-30"],
        start_time="00:00:00",
        end_time="23:59:59",
        version="006",
    )
```